whois-commons/src/main/resources/nrtm_schema.sql

```diff
@@ -9,8 +9,24 @@
 /*!40101 SET @OLD_SQL_MODE = @@SQL_MODE, SQL_MODE = 'NO_AUTO_VALUE_ON_ZERO' */;
 /*!40111 SET @OLD_SQL_NOTES = @@SQL_NOTES, SQL_NOTES = 0 */;
 
+DROP TABLE IF EXISTS `version`;
+CREATE TABLE `version`
+(
+    `version` varchar(80)
+) ENGINE = InnoDB
+  DEFAULT CHARSET = utf8;
+
+INSERT INTO version
```

Comment:
Put INSERTs and UPDATEs into a separate `nrtm_data.sql` file for consistency.

Reply:
done